types/firestore.d.ts

```diff
@@ -1396,7 +1394,10 @@ declare namespace FirebaseFirestore {
      * `null` removes the current converter.
      * @return A DocumentReference<U> that uses the provided converter.
      */
-    withConverter<NewAppModelType, NewDbModelType extends DocumentData>(
+    withConverter<
```

Comment:
Please update the definitions of `withConverter()` functions in the implementation files collection-group.ts and reference.ts as well, so that they are consistent with the public signature.